plugins/modules/floating_ip.py

```diff
@@ -15,30 +15,38 @@
 description:
   - Create, assign and delete floating IPs on the cloudscale.ch IaaS service.
 notes:
-  - Idempotency depends on the C(ip) parameter.
   - Once a floating_ip is created, all parameters except C(server), C(reverse_ptr) and C(tags) are read-only.
   - This module requires the ipaddress python library. This library is included in Python since version 3.3. It is available as a
     module on PyPI for earlier versions.
 author:
   - Gaudenz Steinlin (@gaudenz)
   - Denis Krienbühl (@href)
+  - René Moser (@resmo)
 version_added: 1.0.0
 options:
+  network:
+    description:
+      - Floating IP address to change.
+      - One of C(network) or C(name) is required to identify the floating IP.
```

Comment:
```suggestion
      - One of I(network) or I(name) is required to identify the floating IP.
```
`I(...)` is for option names, `C(...)` for values.